backend/src/Program.cs

```diff
@@ -127,12 +127,12 @@ IServiceProvider services
         }
     }
 
-    private static async Task ImportGnuPGKey(
+    private static async Task InitializeSigningService(
         IServiceProvider services
     )
     {
         var signing = services.GetRequiredService<ISigningService>();
-        await signing.ImportPrivateKey();
+        await signing.Initialize();
```

Comment:
Currently `Initialize` returns a boolean for success or failure. It is being ignored here. What happens later on if the Webserver runs even though this initialization failed? I would throw an exception either here when the return value is false or directly throw an exception in the method `Initialize` instead of returning a boolean. Personally I don't like booleans to communicate success or failure because they don't convey information about what went wrong, where things went wrong and so forth.